### DISCUSSION TITLE
Filter out non user entered comments on isssues

### DISCUSSION THREAD

**Discussion by dcanning** [community/community#4579] [Projects and Issues]
actions taken on issues can become quite long, it would be nice to have a way to filter out things that are not actual comments

![image](https://user-images.githubusercontent.com/1880970/125095148-e28a4100-e0a1-11eb-93ea-cab3441d9785.png)

**Comment by jpwegmuller50** [comment]
Having a toggle to show only comments or all timeline events would be great.  It would allow a "cleaner" view for just reviewing comments.
